main/src/main.rs

```diff
@@ -77,25 +74,28 @@ enum SubCommand {
     /// run cnosdb server
     #[clap(arg_required_else_help = true)]
     Run {},
-    /// run tskv
```

Comment:
The doc https://github.com/cnosdb/cnosdb/blob/main/docs/get-started.md#run needs to be updated.